buildkite/scripts/build-hardfork-package.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -eo pipefail
+
+([ -z "$DUNE_PROFILE" ] || [ -z "$CONFIG_JSON_GZ_URL" ] || [ -z "$MINA_DEB_CODENAME" ]) && echo "required env vars were not provided" && exit 1
+
+source ~/.profile
+
+MINA_COMMIT_SHA1=$(git rev-parse HEAD)
+
+echo "--- Download and extract previous network config"
+curl -o config.json.gz $CONFIG_JSON_GZ_URL
+gunzip config.json.gz
+
+echo "--- Migrate accounts to new network format"
+# TODO: At this stage, we need to migrate the json accounts into the new network's format.
+#       For now, this is hard-coded to the mainnet -> berkeley migration, but we need to select
+#       a migration to perform in the future.
+# NB: we use sed here instead of jq, because jq is extremely slow at processing this file
+sed -i -e 's/"set_verification_key": "signature"/"set_verification_key": {"auth": "signature", "txn_version": "1"}/' config.json
+
+echo "--- Build libp2p_helper"
+make -C src/app/libp2p_helper
+
+MAINNET_TARGETS=""
+[[ ${MINA_BUILD_MAINNET} ]] && MAINNET_TARGETS="src/app/cli/src/mina_mainnet_signatures.exe src/app/rosetta/rosetta_mainnet_signatures.exe src/app/rosetta/ocaml-signer/signer_mainnet_signatures.exe"
+
+echo "--- Build all major tagets required for packaging"
```

Comment:
This duplicated build-artifacts.sh script. Was there any reason not to reuse this part? Are all binaries needed for hardfor package generation?